README.md

```diff
@@ -1239,203 +794,63 @@ It will start in the current working directory, or whatever is specified with `-
-        stdout:    /var/folders/lt/b3cs9y610fg_13q14dckwcvm0000gn/T/pytest_workflow_ahvulf1v/fastqc_paired-end/log.out
-        stderr:    /var/folders/lt/b3cs9y610fg_13q14dckwcvm0000gn/T/pytest_workflow_ahvulf1v/fastqc_paired-end/log.err
-'fastqc paired-end' done.
-
-tests/test_versions_yml.py sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 17%]
-ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 38%]
-ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 59%]
-sssssssssssssssssssssssssssssssssssssssssssssssssssssssssss..ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss ssss [ 80%]
-ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss     [ 98%]
-tests/modules/fastqc/test.yml ........
-Keeping temporary directories and logs. Use '--kwd' or '--keep-workflow-wd' to disable this behaviour.
-=================================================== 10 passed, 751 skipped, 479 warnings in 50.76s ===================================================
-```
+<!-- RICH-CODEX
+working_dir: tmp/modules
+timeout: 30
+extra_env:
+  PROFILE: 'conda'
+-->
+
+![`nf-core modules test samtools/view --no-prompts`](docs/images/nf-core-modules-test.svg)
 
 ### Bump bioconda and container versions of modules in
 
 If you are contributing to the `nf-core/modules` repository and want to bump bioconda and container versions of certain modules, you can use the `nf-core modules bump-versions` helper tool. This will bump the bioconda version of a single or all modules to the latest version and also fetch the correct Docker and Singularity container tags.
 
-```console
-$ nf-core modules bump-versions -d modules
-
-                                          ,--./,-.
-          ___     __   __   __   ___     /,-._.--~\
-    |\ | |__  __ /  ` /  \ |__) |__         }  {
-    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
-                                          `._,._,'
-
-    nf-core/tools version 2.2
-
+<!-- RICH-CODEX
+working_dir: tmp/modules
+-->
 
-
-? Bump versions for all modules or a single named module?  Named module
-? Tool name: bcftools/consensus
-╭───────────────────────────╮
-│ [!] 1  Module updated     │
-╰───────────────────────────╯
-╭─────────────────────────────────────────────────────────────╮
-│ Module name               │ Update message                  │
-├───────────────────────────┤─────────────────────────────────┤
-│ bcftools/consensus        │ Module updated:  1.11 --> 1.12  │
-╰─────────────────────────────────────────────────────────────╯
-```
+![`nf-core modules bump-versions multiqc`](docs/images/nf-core-modules-bump-version.svg)
```

Comment:
The update of the module is failing, is this expected?

Reply:
I will try to fix that.

Reply:
done [2eed3a1](https://github.com/nf-core/tools/pull/1764/commits/2eed3a1eb3a8b4d09fc3a6840b1695c28be330ca)